mainnet-1/specs/movementt.json

```diff
@@ -0,0 +1,110 @@
+{
+    "proposal": {
+        "title": "Add Specs: Movement Testnet",
+        "description": "Adding new specification support for relaying Movement testnet bardock data on Lava",
+        "specs": [
+            {
+                "index": "MOVEMENTT",
+                "name": "movement testnet bardock",
+                "enabled": true,
+                "imports": [
+                    "APT1"
+                ],
+                "reliability_threshold": 268435455,
+                "data_reliability_enabled": true,
+                "block_distance_for_finalized_data": 5,
+                "blocks_in_finalization_proof": 5,
+                "average_block_time": 1000,
+                "allowed_block_lag_for_qos_sync": 50,
+                "shares": 1,
+                "min_stake_provider": {
+                    "denom": "ulava",
+                    "amount": "5000000000"
+                },
+                "api_collections": [
+                    {
+                        "enabled": true,
+                        "collection_data": {
+                            "api_interface": "rest",
+                            "internal_path": "",
+                            "type": "GET",
+                            "add_on": ""
+                        },
+                        "apis": [
+                            {
+                                "name": "/transactions/wait_by_hash/{txn_hash}",
+                                "block_parsing": {
+                                    "parser_arg": [
+                                        "latest"
+                                    ],
+                                    "parser_func": "DEFAULT"
+                                },
+                                "compute_units": 10,
+                                "enabled": true,
+                                "category": {
+                                    "deterministic": true,
+                                    "local": false,
+                                    "subscription": false,
+                                    "stateful": 0
+                                },
+                                "extra_compute_units": 0
+                            }
+                        ],
+                        "headers": [
+                            {
+                                "name": "x-aptos-gas-used",
+                                "kind": "pass_reply",
+                                "function_tag": "DISABLED"
```

Comment:
why is this header disabled?

Reply:
we need enable all ? 
![Screenshot 2024-11-14 at 20 01 50](https://github.com/user-attachments/assets/b27ebfe4-6dea-4bc6-815c-fea3357e6080)

Reply:
if you inherit it from aptos than you dont need to do it again